types/chat.ts

```diff
@@ -190,6 +191,12 @@ export interface ChatOptions {
      * Server signals to Chat Client support of Chat export feature.
      */
     export?: boolean
+
+    /*
+        Server signals to Chat Client support of Chat notifications.
+        Currently only supports chat notifications for developer profiles.
```

Comment:
Why is it? The type is just `ChatMessage` - we could put any text into the `body` there. "developer profiles" - this is how we plan to use it, but I do not think the protocol limits us to this use case.